src/core/utils/useReservableFromAnotherLibrary.tsx

```diff
@@ -13,7 +13,7 @@ const useReservableFromAnotherLibrary = (
   const config = useConfig();
   const { data: holdingsData } = useGetHoldings({
     faustIds: getAllFaustIds(manifestations),
-    useAvailabilityBlacklist: true,
+    blacklist: "both",
```

Comment:
:+1: `useAvailabilityBlacklist: true` was mapped to the `both` type. Now it is the `availability` type. Behavior should be unchanged.